scripts/js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "js-scripts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "runtimeUpgrade": "ts-node ./src/runtimeUpgrade.ts",
+    "lint": "eslint -c .eslintrc.js --ext .ts ./src",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "typecheck": "tsc --noEmit"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@polkadot/api": "^8.3.2",
+    "@types/jest": "^27.5.0",
+    "@types/ws": "^8.5.3",
+    "@typescript-eslint/eslint-plugin": "^5.22.0",
+    "@typescript-eslint/parser": "^5.22.0",
+    "axios": "^0.27.2",
+    "creditcoin-js": "file:../../creditcoin-js/lib",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "^8.5.0",
+    "ethers": "^5.6.5",
+    "jest": "^28.0.3",
+    "js-guid": "^1.0.0",
+    "prettier": "^2.6.2",
+    "ts-jest": "^28.0.1",
+    "ts-node": "^10.7.0",
+    "typescript": "^4.6.4",
+    "ws": "^8.5.0"
```

Comment:
This path needs to be added to dependabot config as well.

 (and deal with the dependencies like we discussed yesterday but outside of this PR).